docs/node/run-your-node/validator-node/README.md

```diff
@@ -88,13 +88,22 @@ mkdir -m700 -p {entity,node}
 
 ### Copying the Genesis File
 
-The latest genesis file can be found in [Network Parameters](../../mainnet/README.md). You should download the latest `genesis.json` file, copy it to the working directory and set the following environment variable pointing to its path:
+To continue, download the most recent `genesis.json` file from the
+[Network Parameters] section and save it to your `/localhostdir` working
+directory:
+
+```bash
+cd /localhostdir
+curl https://github.com/oasisprotocol/mainnet-artifacts/releases/download/2022-04-11/genesis.json
```

Comment:
Did you test this?

```suggestion
curl -L -o genesis.json https://github.com/oasisprotocol/mainnet-artifacts/releases/download/2022-04-11/genesis.json
```

Also, I wouldn't put literal dates here, because they will get obsolete. Perhaps something like:

```suggestion
curl -L -o genesis.json https://github.com/oasisprotocol/mainnet-artifacts/releases/download/RELEASE_DATE/genesis.json # replace RELEASE_DATE with actual Mainnet consensus dump date, e.g. 2022-11-04
```